### PR TITLE
Create bfe-opentracing

### PR DESCRIPTION
**Pull request by iyangsj**
Add bfe project to the registry

### CODE REVIEW

File: content/registry/bfe-opentracing [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: bfe-opentracing
+registryType: instrumentation
+tags:
+  - opentracing
+  - bfe
+  - golang
+  
+repo: https://github.com/baidu/bfe/tree/master/bfe_modules/mod_trace
```
Comment (yurishkuro) [inline content/registry/bfe-opentracing:9]: I suggest adding a README there, there seems to be no documentation how to use this module.

File: content/registry/bfe-opentracing [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: bfe-opentracing
+registryType: instrumentation
+tags:
+  - opentracing
+  - bfe
+  - golang
```
[the remark this answers is not in the document]
Comment (iyangsj) [reply content/registry/bfe-opentracing]: Done

File: content/registry/bfe-opentracing [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
```
[the remark this answers is not in the document]
Comment (iyangsj) [reply content/registry/bfe-opentracing]: Done

File: content/registry/bfe-opentracing (hunk 1 above)
Comment (iyangsj) [reply content/registry/bfe-opentracing]: Done

**Pull request merged**